contracts/Launchpeg.sol

```diff
@@ -400,23 +368,19 @@ contract Launchpeg is BaseLaunchpeg, ILaunchpeg {
     function currentPhase() public view override returns (Phase) {
         if (
             auctionSaleStartTime == 0 ||
-            mintlistStartTime == 0 ||
-            publicSaleStartTime == 0 ||
-            block.timestamp < auctionSaleStartTime
+            allowlistStartTime == 0 ||
+            publicSaleStartTime == 0
         ) {
             return Phase.NotStarted;
-        } else if (
-            block.timestamp >= auctionSaleStartTime &&
-            block.timestamp < mintlistStartTime
-        ) {
+        }
+        if (block.timestamp >= publicSaleStartTime) {
+            return Phase.PublicSale;
+        } else if (block.timestamp >= allowlistStartTime) {
+            return Phase.Allowlist;
+        } else if (block.timestamp >= auctionSaleStartTime) {
             return Phase.DutchAuction;
-        } else if (
-            block.timestamp >= mintlistStartTime &&
-            block.timestamp < publicSaleStartTime
-        ) {
-            return Phase.Mintlist;
         }
-        return Phase.PublicSale;
+        return Phase.NotStarted;
```

Comment:
Curious to know what was wrong with the other way of doing the if-elses here.

Reply:
It was a comment from Maxime, it feels slightly more readable like this

Reply:
yeah i thought it was easier to read but if you prefer to keep the other way because it's what was audited, np